WebCryptoAPI/generateKey/failures.js

```diff
@@ -0,0 +1,219 @@
+function run_test(algorithmNames) {
+    // API may not be available outside a secure context.
+    if (!runningInASecureContext()) {
+        test(function() {}, "No tests because API not necessarily available in insecure context");
```

Comment:
When we talked you said you were not using the runningInASecureContext function.  Should this be removed?